common/devices.go

```diff
@@ -4,9 +4,14 @@ import "runtime"
 
 const (
 	AndroidPlatform = "android"
+	IosPlatform     = "ios"
```

Comment:
I don't know how much we follow the conventions from here https://google.github.io/styleguide/go/decisions#initialisms, but if we do it would be `IOSPlatform`